### PR TITLE
Add OCM server metrics to perf ES

### PR DESCRIPTION
**Pull request by venkataanil**
OCM server consists of cluster service and account manager service.
During ocm load testing we monitor metrics from both these services.
In this change, kube-burner pulls metrics from ocm prometheus server
for both the services and index them in perf ES accordingly.

We are using the same prometheus queries which OCM team is using
in their internal grafana dashboards.

Grafana dashboard for clusters service [1] and account manager
service [2] in our perf ES
[1] https://grafana.apps.observability.perfscale.devcluster.openshift.com/d/uhc-clusters-service/uhc-clusters-service?orgId=1&var-datasource=ocm-uhc-clusters-service&var-uuid=a5ed9dce-d7be-40a7-a196-1a5da349dbec&from=1657606690358&to=1657621572537
[2] https://grafana.apps.observability.perfscale.devcluster.openshift.com/d/uhc-account-manager/uhc-account-manager?orgId=1&from=1657607400000&to=1657621800000

ES docs for clusters service [3] and account manager service [4] in
our perf ES
[3] https://perf-results-kibana.apps.observability.perfscale.devcluster.openshift.com/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-07-12T06:30:00.000Z',to:'2022-07-12T11:00:00.000Z'))&_a=(columns:!(),filters:!(),index:'9b6b77e0-fe90-11ec-84f5-13fd83e5c2e7',interval:auto,query:(language:kuery,query:'uuid.keyword%20:%20%22a5ed9dce-d7be-40a7-a196-1a5da349dbec%22%20'),sort:!(!(timestamp,desc)))
[4] https://perf-results-kibana.apps.observability.perfscale.devcluster.openshift.com/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-07-12T06:30:00.000Z',to:'2022-07-12T11:00:00.000Z'))&_a=(columns:!(),filters:!(),index:c43ced70-fc60-11ec-b5e5-e5f784bcd40c,interval:auto,query:(language:kuery,query:'uuid.keyword%20:%20%22a5ed9dce-d7be-40a7-a196-1a5da349dbec%22%20'),sort:!(!(timestamp,desc)))

Original internal OCM server grafna dashboards (Maintained by OCM team)
https://grafana.app-sre.devshift.net/d/uhc-account-manager/uhc-account-manager?orgId=1&from=now-30m&to=now&var-datasource=app-sre-stage-01-prometheus&var-namespace=uhc-integration&var-rds_datasource=app-sre-stage-01-prometheus&var-rds_namespace=uhc-acct-mngr-integration&refresh=30s
https://grafana.app-sre.devshift.net/d/uhc-clusters-service/uhc-clusters-service?orgId=1&var-datasource=app-sre-stage-01-prometheus&var-namespace=uhc-stage&var-pod=All

### CODE REVIEW

File: workloads/api-load/env.sh [hunk 1]
```diff
@@ -4,6 +4,11 @@ TEST_CLEANUP=${TEST_CLEANUP:-true}
 export ES_SERVER=${ES_SERVER:-https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443}
 export ES_INDEX=ripsaw-api-load
```
Comment (chentex) [inline workloads/api-load/env.sh:5]: I know this is not part of the PR, but are we still usign the same index for `ripsaw` and for our results?

**Review by chentex — APPROVED** [review]
LGMT

File: workloads/api-load/env.sh (hunk 1 above)
Comment (venkataanil) [reply workloads/api-load/env.sh:5]: I think we agreed for ocm-request-test. If you confirm, I will update this here.

Comment (venkataanil) [reply workloads/api-load/env.sh:5]: Thanks, I will push a separate PR for that.

**Review by jtaleric — APPROVED** [review]
lgtm

**Review by mukrishn — APPROVED** [review]
LGTM

**Comment by smalleni** [comment]
@mukrishn @chentex why is this still sitting here?

**Comment by mukrishn** [comment]
Not sure why is it still open @venkataanil any more changes expected ?

**Pull request closed without merging**